addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -9,7 +9,7 @@
                         <i class="fa fa-arrow-left"/>
                     </div>
                 </t>
-                <t t-if="chatWindow.thread">
+                <t t-if="chatWindow.thread and chatWindow.thread.channel_type">
```

Comment:
Using `chatWindow.thread.model === 'mail.channel'` is conceptually better than `channel_type` (which should only be used in the context of channel) if you want to exclude non-channels.

You will need to add dependency on `thread.model` in the `useStore` of the component too (just to be safe, but model of thread shouldn't change normally).

Reply:
Ok.. Changes are done.! Thanks.